UltraStar Play/Assets/Common/Util/ImageManager.cs

```diff
@@ -0,0 +1,29 @@
+
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+public class ImageManager
+{
+
+    private static readonly Dictionary<string, Sprite> spriteCache = new Dictionary<string, Sprite>();
```

Comment:
This Dictionary should probably be changed to be size-limited and use for example least frequently used as cache replacement policy, once the available application memory gets low. If this cache is used also for song cover images, it can get quite big depending on how many songs there are and depending on the image format and size.
Could you please change this to [MemoryCache ](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.caching.memorycache?redirectedfrom=MSDN&view=netframework-4.8)or similar?